SPECS/moby-containerd-cc/containerd.service

```diff
@@ -0,0 +1,14 @@
+[Unit]
```

Comment:
On a general note, I believe we are doing some containerd-cc workaround in the kata-containers-cc package - I think we would want to straighten this out as part of this PR?

Reply:
The override has been removed from kata along with its symlink